src/tests/ftest/erasurecode/mdtest_smoke.py

```diff
@@ -16,7 +16,7 @@ class EcodMdtest(MdtestBase):
     :avocado: recursive
     """
 
-    def test_mdtest_large(self):
+    def test_ec_mdtest_large(self):
```

Comment:
it's not actually that large and it's just smoke test so may be change it to smoke?

```suggestion
    def test_ec_mdtest_smoke(self):
```